js/views/callinfoview.js

```diff
@@ -203,6 +203,12 @@
 			var roomURL = OC.generateUrl('/call/' + this.model.get('token')),
 				completeURL = window.location.protocol + '//' + window.location.host + roomURL;
 
+			this.ui.lobbyTimerInput.tooltip({
+				placement: 'bottom',
+				trigger: 'hover',
+				title: t('spreed', 'YYYY-MM-DD HH:MM')
```

Comment:
```suggestion
				title: 'YYYY-MM-DD HH:mm'
```

Minutes are lowercase mm also this should not be translated, or do we actually parse it in different languages?